<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Disciplines</class>
 <widget class="QWidget" name="Disciplines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="Discipline1" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#Discipline1{
	background-color: rgb(85, 170, 127);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Discipline1_0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator{

width: 15px;
height: 15px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_25">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>23</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DisciplineButtonsLayout_1">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QRadioButton" name="disciplineButton1_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">disciplineButtonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disciplineButton1_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">disciplineButtonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disciplineButton1_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">disciplineButtonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disciplineButton1_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">disciplineButtonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="disciplineButton1_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">disciplineButtonGroup1</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="DisciplinePowers1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="newButton">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="disciplineButtonGroup1">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
